src/microbe_stage/Spawners.cs

```diff
@@ -58,6 +58,163 @@ public static class SpawnHelpers
+                        curSpawn.x += random.Next(-2, 3);
+
+                        entities.Add(SpawnMicrobe(species, location + curSpawn, worldRoot, microbeScene, true,
+                            cloudSystem, currentGame));
+                    }
+                }
+                else if (random.Next(0, 5) < 2 && !horizontal)
+                {
+                    horizontal = true;
+                    vertical = false;
+
+                    for (int c = 0; c < random.Next(Constants.MIN_BACTERIAL_LINE_SIZE,
+                        Constants.MAX_BACTERIAL_LINE_SIZE + 1); c++)
+                    {
+                        // Dont spawn them on top of each other because
+                        // It causes them to bounce around and lag
+                        curSpawn.x -= random.Next(5, 8);
+
+                        // Add a litlle organicness to the look
+                        curSpawn.z -= random.Next(-2, 3);
+
+                        entities.Add(SpawnMicrobe(species, location + curSpawn, worldRoot, microbeScene, true,
+                            cloudSystem, currentGame));
+                    }
+                }
+                else if (random.Next(0, 5) < 2 && !vertical)
+                {
+                    horizontal = false;
+                    vertical = true;
+
+                    for (int c = 0; c < random.Next(Constants.MIN_BACTERIAL_LINE_SIZE,
+                        Constants.MAX_BACTERIAL_LINE_SIZE + 1); c++)
+                    {
+                        // Dont spawn them on top of each other because it
+                        // causes them to bounce around and lag
+                        curSpawn.z -= random.Next(5, 8);
+
+                        // Add a litlle organicness to the look
+                        curSpawn.x -= random.Next(-2, 3);
+
+                        entities.Add(SpawnMicrobe(species, location + curSpawn, worldRoot, microbeScene, true,
+                            cloudSystem, currentGame));
+                    }
+                }
+                else
+                {
+                    // Diagonal
+                    horizontal = false;
+                    vertical = false;
+
+                    for (int c = 0; c < random.Next(Constants.MIN_BACTERIAL_LINE_SIZE,
+                        Constants.MAX_BACTERIAL_LINE_SIZE + 1); c++)
+                    {
+                        // Dont spawn them on top of each other because it
+                        // Causes them to bounce around and lag
+                        curSpawn.z += random.Next(5, 8);
+
+                        curSpawn.x += random.Next(5, 8);
+
+                        entities.Add(SpawnMicrobe(species, location + curSpawn, worldRoot, microbeScene, true,
+                            cloudSystem, currentGame));
+                    }
+                }
+            }
+        }
+
+        entities.Add(SpawnMicrobe(species, location, worldRoot, microbeScene, true, cloudSystem, currentGame));
```

Comment:
Doesn't this spawn a cell exactly on top of what the spawn function has spawned just beforehand?
So I think this line should be removed.